tests/classes_unit_tests/testCanonicalHyperCubeSetNew.py

```diff
@@ -971,6 +973,33 @@ def test_contained_in_new(self):
         d = CanonicalHyperCubeSet.create_from_cube(dimensions, [get_str_dfa("x|y|z")], ["paths"])
         self.assertTrue(c.contained_in(d))
 
+    def test_bug_in_contained(self):
+        BasePeerSet.reset()
+        BasePeerSet().add_peer("A")
+        BasePeerSet().add_peer("B")
+        BasePeerSet().add_peer("C")
+        ipv4_block = IpBlock.get_all_ips_block(True)
+        my_dimensions1 = ["src_peers", "dst_peers"]
+        my_dimensions2 = ["src_peers", "dst_peers", "protocols"]
+        conns1 = CanonicalHyperCubeSet.create_from_cube(my_dimensions1,
```

Comment:
why is `my_dimensions1` the first arg (all_dims) to `create_from_cube`, and not `my_dimensions2`?

Reply:
My mistake. Changed to my_dimensions2.